### PR TITLE
Update configandcategorize - Supplier info.md

### PR DESCRIPTION
**Pull request by Amy-Roberts**
# Description

* Include a summary of the new content you're adding or the reason for the change. 
* Include relevant context.
* Do not link to work items.
* Any PNG images should be compressed using [Tiny PNG](https://tinypng.com/) or equivalent.

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/sync-for-expenses/configandcategorize.md [hunk 1]
```diff
@@ -71,10 +71,10 @@ Sync for Expenses currently supports option 2 only.
 </ul>
 
 - Suppliers can be set at the configuration level and also at the [transaction level](https://docs.codat.io/sync-for-expenses-api#/operations/create-expense-dataset#request-body).
-    - By setting the supplier at the transaction level you will be able to override the configuration supplier, in order to sync a more accurate   
+    - By setting the supplier at the transaction level you will be able to override the configuration supplier, in order to sync to a more accurate
```
Comment (pzaichkina) [inline docs/sync-for-expenses/configandcategorize.md:74]:
```suggestion
    - By setting the supplier at the transaction level, you will be able to override the configuration supplier in order to sync to a more accurate representation of who the spend should be associated with in the accounting platform.
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 2]
```diff
@@ -71,10 +71,10 @@ Sync for Expenses currently supports option 2 only.
 </ul>
 
 - Suppliers can be set at the configuration level and also at the [transaction level](https://docs.codat.io/sync-for-expenses-api#/operations/create-expense-dataset#request-body).
-    - By setting the supplier at the transaction level you will be able to override the configuration supplier, in order to sync a more accurate   
+    - By setting the supplier at the transaction level you will be able to override the configuration supplier, in order to sync to a more accurate   
       representation of who the spend should be associated with in the accounting platform.
```
Comment (pzaichkina) [inline docs/sync-for-expenses/configandcategorize.md:75]:
```suggestion
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 3]
```diff
@@ -71,10 +71,9 @@ Sync for Expenses currently supports option 2 only.
 </ul>
 
 - Suppliers can be set at the configuration level and also at the [transaction level](https://docs.codat.io/sync-for-expenses-api#/operations/create-expense-dataset#request-body).
-    - By setting the supplier at the transaction level you will be able to override the configuration supplier, in order to sync a more accurate   
-      representation of who the spend should be associated with in the accounting platform. 
+    - By setting the supplier at the transaction level, you will be able to override the configuration supplier in order to sync to a more accurate representation of who the spend should be associated with in the accounting platform.
     - If no supplier is set at the transaction level, the spend will have the config supplier set as a default against it. 
-    - The currency associated with the supplier must match the currency associated with the spend.
+    - The currency associated with the supplier must match the currency associated with the spend. This is validated by Codat, but does not validate against suppliers which allow for multiple currencies.
```
Comment (pzaichkina) [inline docs/sync-for-expenses/configandcategorize.md:76]: do i understand this right? codat validates that supplier currency matches spend currency, but if a supplier supports many currencies, then codat does not validate against spend currency?

Comment (pzaichkina) [reply docs/sync-for-expenses/configandcategorize.md:76]: because if so, we should reword the sentence to make it clearer

Comment (Amy-Roberts) [reply docs/sync-for-expenses/configandcategorize.md:76]: Yeah you can sometimes set a supplier in an accounting platform to work with multiple currencies. We don't validate for this. We only work with supplier which have one set currency.

Comment (pzaichkina) [inline docs/sync-for-expenses/configandcategorize.md:76]:
```suggestion
    - The currency associated with the supplier must match the currency associated with the spend. Codat validates the match for suppliers with a single set currency, but not for suppliers that work with multiple currencies.  
```

**Review by pzaichkina — APPROVED** [review]

**Pull request closed without merging**